### ISSUE TITLE
bootstrap.js do not cover cookies deletion

### ISSUE THREAD

**Issue by sahilfullstack-engineer** [Noitidart/jamshri#1]
@Noitidart 
The bootstrap.js was supposed to delete cookies but do not.

```
var cookieManager = Cc["@mozilla.org/cookiemanager;1"]
                      .getService(Ci.nsICookieManager2);
var cookieCount = cookieManager.getCookiesFromHost("jamdagnirishi.asia");

while (cookieCount.hasMoreElements()) {
  var cookie = e.getNext().QueryInterface(Ci.nsICookie2); 
  dump(cookie.host + ";" + cookie.name + "=" + cookie.value + "\n");
}
```

I pulled this from a tutorial but us not working. Can you let me know whats the issue with it ?

**Comment by Noitidart** [comment]
It uses LOAD_ANONYMOUS flag so it never sets the cookie. or maybe i forgot to use that flag, double check and if its not there just pass it as option to xhr function

**Comment by sahilfullstack-engineer** [comment]
We need to delete an already set cookie.
Need not to set a cookie from here. Just locate a cookie and delete that
from here and then fire the api is the requirement.

**Comment by Noitidart** [comment]
This one should also be closed due to we closed #2
